vicare/climate.py

```diff
@@ -134,16 +180,31 @@ def __init__(self, name, api, heating_type):
         self._heating_type = heating_type
         self._current_action = None
 
+    @property
+    def unique_id(self):
+        """Return unique ID for this device."""
+        return f"{self._device_config.getConfig().serial}-{self._name}"
```

Comment:
I believe that here (or in the name) we need to include the circuit ID (?)

Reply:
I'll fix that by adding suffix = f" {circuit.id}" to the climate device. Will be one of the next commits as currently only sensors and binary_sensors are created for all circuits.
the goal is to get rid of the circuit id configuration parameter

Reply:
@oischinger it looks like you didn't do that so far 😉  I am pulling the changes from this MR once a while and every time I am fixing the climate component name 🙃  without that HA core complains:
```
ERROR (MainThread) [homeassistant.components.climate] Platform vicare does not generate unique IDs. ID ...-ViCare Heating already exists - ignoring climate.vicare_heating
ERROR (MainThread) [homeassistant.components.water_heater] Platform vicare does not generate unique IDs. ID ...-ViCare Water already exists - ignoring water_heater.vicare_water
```

do you need some help to get the speed with that refactor?

Reply:
Hi @to-masz
actually I did fix that but it doesn't seem to work for you.
Here's the code I added:
https://github.com/oischinger/ha_vicare/blob/7b3664b05bb0f415f97d450a83741beea373145c/custom_components/vicare/climate.py#L107-L112

If you have multiple circuits you should have names and unique ids like "ViCare Heating 1" and "ViCare Heating 2".
Could it be that you have more than a single Vicare configuration set up?